Source/Plugins/Core/com.equella.core/js/tsrc/modules/SearchModule.ts

```diff
@@ -221,18 +366,35 @@ export const searchItems = ({
   return OEQ.Search.search(API_BASE_URL, searchParams);
 };
 
+const getCollectionDetails = async (
```

Comment:
As discussed, let's tweak this name a bit. At first I was going to suggest `getCollectionsByUuid` however this is just a find, as anything that doesn't match is silently skipped. So:

```suggestion
const findCollectionsByUuid = async (
```